R/bench_standardize.R

```diff
@@ -1,142 +1,131 @@
-                "log_time")
->>>>>>> main
-
-
- # summary for each dataset ----
-
- mean_se <- bm_comb %>%
-  group_by(rfvs, dataset) %>%
-  summarize(across(.cols = all_of(smry_cols),
-                   .fns = list(mean = ~ mean(.x, na.rm = TRUE),
-                               se = ~ sd(.x) / (length(.x) - 1))))
-
- quants <- bm_comb %>%
-  reframe(
-   across(
-    .cols = c(n_selected, perc_reduced,
-              rmse_axis, rsq_axis,
-              rmse_oblique, rsq_oblique,
-              time, log_time,
-              rsq_axis_z, rsq_oblique_z,
-              time_z,log_time_z, perc_reduced_z),
-    .fns = ~ quantile(.x, probs = c(1,2,3) / 4, na.rm = TRUE)
-   ),
-   quantile = c(25, 50, 75),
-   .by = c(rfvs, dataset)
-  ) %>%
-  pivot_wider(names_from = quantile,
-              values_from = all_of(smry_cols))
-
- smry_by_data <- left_join(mean_se, quants)
-
-<<<<<<< HEAD
- # summary overall -----
-
- mean_se <- bm_comb %>%
-  group_by(rfvs) %>%
-  summarize(across(.cols = all_of(smry_cols),
-                   .fns = list(mean = ~ mean(.x, na.rm = TRUE),
-                               se = ~ sd(.x) / sqrt((length(.x) - 1)))))
-
- quants <- bm_comb %>%
-  reframe(
-   across(
-    .cols = c(n_selected, perc_reduced,
-              rmse_axis, rsq_axis,
-              rmse_oblique, rsq_oblique,
-              time, log_time,
-              rsq_axis_z, rsq_oblique_z,
-              time_z,log_time_z, perc_reduced_z),
-    .fns = ~ quantile(.x, probs = c(1,2,3) / 4, na.rm = TRUE)
-   ),
-   quantile = c(25, 50, 75),
-   .by = c(rfvs)
-  ) %>%
-  pivot_wider(names_from = quantile,
-              values_from = all_of(smry_cols))
-
- smry_overall <- left_join(mean_se, quants) %>%
-  mutate(dataset = 'overall', .after = rfvs)
-
- #create new variable called method with long form label name for each VS method
- lab_names <- c("Altman", "Menze", "Permute -\nOblique", "Boruta", "CARET", "Hapfelmeier", "Jiang", "Min Depth \nMedium",
-                "Negation", "None", "Permute - \nAxis", "RRF", "Svetnik", "VSURF")
- smry_by_data$method <- factor(smry_by_data$rfvs, levels(factor(smry_by_data$rfvs)), lab_names[all_rfvs %in% levels(factor(smry_by_data$rfvs))])
- smry_overall$method <- factor(smry_overall$rfvs, levels(factor(smry_overall$rfvs)), lab_names[all_rfvs %in% levels(factor(smry_overall$rfvs))])
-
-
- list(by_data = smry_by_data,
-      overall = smry_overall)
-=======
- as.data.frame(smry_by_data)
-
->>>>>>> main
-}
-
-
+
+##Function creates z-scores for for each dataset across methods and iterations, and then averages them
+# ignore is a vector of methods to ignore in computing z-scores
+# Paramater Input:
+#    bm_comb: raw bm_comb data that will be aggregated over
+#    datasets_cv: include if you would like to filter results based on dataset Coefficeint of variation.
+#    cv.thresh: vector of lower and upper boundaries for datasets CV to analyze
+#    ignore: vector of methods to exclude. Relevant for standarized scores
+#

```

Comment:
consider using roxygen formatting

Reply:
It doesn't do anything since this isnt an R package, but it is the format that most R users are familiar with when it comes to reading documentation in code.